packages/cli/src/commands/start.ts

```diff
@@ -60,8 +60,8 @@ interface InstallDependenciesOptions {
 
 /** dependencies for the project */
 async function installServicesAsDependencies({ packageManager, projectAbsolutePath, services }: InstallDependenciesOptions): Promise<void> {
-  // in case service is empty we add dummy package. we use typescript for sure, so we use it.
-  const pkg = ([...services.map(service => serviceToConnector(service)), "typescript"]).join(" ");
+  /* if no services are selected, undefined is passed to the package manager */
+  const pkg = services.length > 0 ? ([...services.map(service => serviceToConnector(service))]).join(" ") : undefined;
```

Comment:
remove dummy `typescript` because we support passing undefined to install command